packages/ui/components/pages/Fuse/FusePoolPage/MarketsList/BorrowApy.tsx

```diff
@@ -0,0 +1,22 @@
+import { Text, VStack } from '@chakra-ui/react';
+import * as React from 'react';
+
+import { useMidas } from '@ui/context/MidasContext';
+import { useColors } from '@ui/hooks/useColors';
+import { MarketData } from '@ui/types/TokensDataMap';
+import { getBlockTimePerMinuteByChainId } from '@ui/utils/networkData';
+
+export const BorrowApy = ({ asset }: { asset: MarketData }) => {
+  const { currentChain, midasSdk } = useMidas();
+  const blocksPerMin = getBlockTimePerMinuteByChainId(currentChain.id);
+  const borrowAPR = midasSdk.ratePerBlockToAPY(asset.borrowRatePerBlock, blocksPerMin);
```

Comment:
`borrowAPR` and `blocksPerMin` could be wrapped in a `useMemo()`.

Reply:
Fixed